crates/red_knot_python_semantic/resources/mdtest/loops/while_loop.md

```diff
@@ -69,3 +69,16 @@ else:
 
 reveal_type(x)  # revealed: Literal[3, 4, 5]
 ```
+
+## Possibly unbound
```

Comment:
This is a regression-test for something that was broken mid-way into this implementation.